examples/lda_hoffman_sparse.d

```diff
@@ -26,7 +26,7 @@ void main(string[] args)
     if (args.length > 1)
         curFolder = args[1];
     else
-        curFolder = thisExePath.dirName;
+        curFolder = getcwd.buildPath("examples");
```

Comment:
temporary build are done in `/tmp/...`, hence I guess `getcwd` is the best we could try?